src/frontend/src/utils/pipes.ts

```diff
@@ -124,3 +124,8 @@ export const dateUndefinedPipe = (date?: Date): string => {
 export const centsToDollar = (cents: number) => {
   return (cents / 100.0).toFixed(2);
 };
+
+/** Displays a refund source as a string "Code - Name" */
+export const refundSourceToCodeName = (refundSource: ClubAccount, refundCode: number) => {
```

Comment:
Change 'refundSourceToCodeName' to 'codeAndRefundSourceName' to make the function's purpose more clear